tacs/mphys/utils.py

```diff
@@ -0,0 +1,66 @@
+import warnings
+import tacs
+
+
+def add_tacs_constraints(model, scenario):
+    """Call this in the configure method to add all TACS constraints functions
+    in a TacsPostcouplingGroup as constraints in an OpenMDAO model. This saves
+    you having to call OpenMDAO's `add_constraint` method for each constraint
+    function yourself.
+
+    This function relies on you having defined the lower and/or upper bounds
+    of the constraints when you created them in your `constraint_setup`
+    function.
+
+    Parameters
+    ----------
+    model : OpenMDAO model
+        Model to add constraints to
```

Comment:
Is the `model` really needed as an input here? It looks like it's only used to get the root proc for printing? Can we just use the comm from the scenario?

Reply:
You need the model because that's what the constraints are added to

Reply:
Oh, can call `add_constraint` on a scenario too? In that case I agree and will remove the model input

Reply:
Ah, I see that now. But you can just add the constraints directly to the subsystem and skip the model. Something like: `constraint.add_constraint(name, equals=lb, linear=True)`

Reply:
Ah, nvm. Beat me to it